Source/WTF/wtf/text/Base64.cpp

```diff
@@ -193,9 +193,9 @@ template<typename T> static std::optional<Vector<uint8_t>> base64DecodeInternal(
                 if (equalsSignCount)
                     return std::nullopt;
                 destination[destinationLength++] = decodedCharacter;
-            } else if (!options.contains(Base64DecodeOptions::IgnoreSpacesAndNewLines) || (!isLatin1(ch) || !isASCIISpace(ch) || (options.contains(Base64DecodeOptions::DiscardVerticalTab) && ch == '\v'))) {
+            // isASCIISpace includes U+000B which https://infra.spec.whatwg.org/#forgiving-base64 excludes
+            } else if (mode != Base64DecodeMode::DefaultIgnoreWhitespace || (!isLatin1(ch) || !isASCIISpace(ch) || ch == '\v'))
```

Comment:
I wasn't sure how to use isHTMLSpace here.

Also, why is the `!isLatin1(ch)` check here? Performance?

Reply:
https://bugs.webkit.org/show_bug.cgi?id=184529#c11 has some context. Given the CSP and SRI failure I should probably reduce the scope of this somewhat, though those failures should be investigated as well as they are shared with Firefox and the specifications are lacking.

Reply:
I have reduced the scope now by not enforcing padding validation for all non-URL cases. I will follow-up separately on CSP/SRI as there are standards-related issues there.